localstripe/resources.py

```diff
@@ -944,6 +944,25 @@ def _api_list_subscriptions(cls, id, **kwargs):
 
         return cls._api_retrieve(id).subscriptions
 
+    @classmethod
+    def _api_list_sources(cls, id, object=None, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        try:
+            if object is not None:
+                assert isinstance(object, str)
```

Comment:
Can you do the change `assert isinstance(object, str)` => `assert type(object) is str` here too?

After that PR is good for me! (Adrien will make the last call).

Reply:
missed that one, done.

Reply:
Thanks!